packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -184,3 +186,70 @@ export const computeVariant = (
     return 'NO_STATE';
   }
 };
+
+/**
+ * Hook to override the avatar persona data of the users
+ * - Calls {@link AvatarPersonaDataCallback} on each render for provided `userIds`
+ * - Returns an array of the same length as `userIds`. Entries in the array may be undefined if there is no data to return.
+ * @private
+ */
+export const useCustomAvatarPersonaData = (
+  userIds: (string | undefined)[],
+  callBack?: AvatarPersonaDataCallback
+): (AvatarPersonaData | undefined)[] => {
+  const [data, setData] = React.useState<(AvatarPersonaData | undefined)[]>([]);
+  useEffect(() => {
+    (async () => {
+      if (!callBack) {
+        const undefinedData: undefined[] = [];
+        for (let i = 0; i < userIds.length; i++) {
+          undefinedData.push(undefined);
```

Comment:
instead of a for loop, can instead do: `Array(<length>).fill(undefined)`

Reply:
TIL.